external_projects/bccm_player/lib/src/widgets/controls/default_controls.dart

```diff
@@ -59,6 +67,20 @@ class DefaultControls extends HookWidget {
       });
     }
 
+    void seekToRelative(int differenceSec) {
+      totalSeekToDurationMs.value += differenceSec * 1000;
+      double newPositionMs = currentMs + totalSeekToDurationMs.value;
+      if (newPositionMs < 0) {
+        newPositionMs = 0;
+      } else if (newPositionMs > duration) {
+        newPositionMs = duration;
+      }
+      forwardRewindDebouncer.run(() {
+        BccmPlayerInterface.instance.seekTo(playerId, newPositionMs);
```

Comment:
- How can we ensure that if you tap 5 times, that it will always skip forward 5*skipInterval?
- How do we ensure that the timeline will instantly reflect your taps?

Maybe reuse the "scrubTo" function, or at least currentScrub, because it's used in the timeline.
In order to get instant feedback, maybe its enough to change the debouncer logic to never debounce the first call (so first run() is instant, but second run() cannot be within _x milliseconds_)?

Reply:
> In order to get instant feedback, maybe its enough to change the debouncer logic to never debounce the first call (so first run() is instant, but second run() cannot be within _x milliseconds_)?

The point of this is to update the current time instantly the first time right?

Reply:
> How can we ensure that if you tap 5 times, that it will always skip forward 5*skipInterval?

This is taken care of by `totalSeekToDurationMs`.
Right now there is not instant update when the slider is moved. Should I change this?